README.md

```diff
@@ -532,7 +532,17 @@ RANDOM_COLOR
 
 // Render every individual mesh in the scene with a random color.
 RANDOM_NODE_COLOR
+
+// Sets a custom color using the customDebugColor call back. 
+CUSTOM_COLOR_MODE
+```
+### .customDebugColor
+
 ```
+customDebugColor: (tile: Tile, child: Object) => void
```

Comment:
`child` should have type `Object3D` and lets use the `js` extension for coloring the code block:

````
```js
code here
```
````

Reply:
Ah thank, I wasn't 100% sure on the type here.